Assets/Scripts/UI/CombatHUD/CombatTutorial.cs

```diff
@@ -0,0 +1,29 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CombatTutorial : MonoBehaviour
+{
+	private bool tutorialOn;
+	private void Awake()
+	{
+		tutorialOn = true;
+	}
+
+	private float timer = 0.0f;
+	public float secondsToDisplay;
+	private void Update()
+	{
+		timer += Time.deltaTime;
```

Comment:
100% just a comment, because this is a totally valid way to do this, but just in case you didn't know, you can also go startTime = Time.time; on awake, and then check the time elapsed by going (Time.time - startTime).
This is a totally valid way of doing it though, just thought I'd let you know in case it'd be helpful c:

Reply:
Ah, I did it that way because it seemed less likely to have An Problem; time stuff can get finicky and if (for example) the player pauses you don't want the tutorial to disappear while they're not looking, etc.  Whereas with updates, it'll not run this update code while the scene is paused? I THINK anyway.  Hopefully.